### DISCUSSION TITLE
Validation error message in notice subType 14 and 19

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#410] [Q&A] [answered]
Good Morning lab !
in notice 14 and 19 if you select as Nature of the contract,  **supplies**  validation get this error : 
**If Procedure Legal Basis (BT-01) value is equal to "Directive 2014/23/EU", then Main Nature (BT-23) must be different from "Supplies".**
But I always set BT-01 to 2137/85,  that is the first in the list.
Anyway watching the field the error is clear .. in this notice (14 and 19 for example) Nature of the contract has to be different from supplies.
What do you think to change this message ?
Bye !
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi Renato,

- The rule is in fact applied based on the Notice Subtype (i.e. 14 or 19).
- Legal Basis 2137/85 is only for Notice Subtype X01 (i.e. European Economic Interest Grouping),
- Restrictions of usable Legal Basis have not yet been implemented to prevent any blocking situation until these get clarified.
From a business perspective, for forms 14 and 19, Directive 23 is the only acceptable Legal Basis for forms 14 and 19 as shown in the Regulation Annex. The message may be reviewed at the same time constraints will be strengthened and when further valid cases are identified.

**Comment by rengit62** [comment]
OK Yves, thanks, very interesting about Legal Basis.
Anyway what I meant, is, as you wrote and as the rule is made, the choice  of the Main Nature (BT-23) depend on the notice subType, not from the Legal Basic, as instead says the error message.

**Reply by YvesJo** [reply]
Thanks Renato for the clarification. The rule was stated early even before the development process started and the concept of notice subtype did not even exist. I agree the wording may need a review.
For any Notice Subtype, there is a single standard directive referenced (cf Regulation Annex) and this is this directive that drives the notice content. Directives are also better known than Notice Subtypes. It appears that, in some cases, some of our internal stakeholders need another legal basis than the standard ones, however these have to comply also with the standard ones, the reference to which, therefore, remains valid. To my knowledge, this shouldn't be the case for directive 23. However some colleagues are currently dealing with the rules statements and their translations. Applicable restrictions on possible legal basis are also being currently dealt with. To conclude, we shall expect an improvement of the rule.
